src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/TrainerGeneratorBase.cs

```diff
@@ -47,7 +48,10 @@ private void Initialize(PipelineNode node)
             {
                 node.Properties.Add("LabelColumnName", "Label");
             }
-            node.Properties.Add("FeatureColumnName", "Features");
```

Comment:
FeatureColumnName gets added for all existing trainer generators. I added IncludeFeatureColumnName which is false for MatrixFactorization trainer generator only.

Reply:
TODO: 
- [ ] Look more closely at the implementation to see why it was expected for all Trainer Generators to have a FeatureColumn. if it is expected then it should also be added for MatrixFactorization trainer generator